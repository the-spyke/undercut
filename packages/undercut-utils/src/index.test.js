import { describe, expect, test } from "@jest/globals";

import * as exports from "./index.js";

describe(`Utils`, () => {
	test(`exports of the main "utils" entry should be stable`, () => {
		expect(Object.keys(exports).sort()).toMatchSnapshot();
	});
});
