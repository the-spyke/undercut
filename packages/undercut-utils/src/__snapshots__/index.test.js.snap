// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exports of the main entry should be stable 1`] = `
Array [
  "Cohort",
  "abort",
  "asObserver",
  "asUnclosable",
  "asc",
  "close",
  "collectProps",
  "compare",
  "composeComparators",
  "createIterable",
  "delay",
  "desc",
  "filterProps",
  "getDoneItem",
  "getIterator",
  "getRecursiveMapper",
  "hasOwnProps",
  "identity",
  "isArrayBuffer",
  "isBoolean",
  "isCoroutine",
  "isDate",
  "isDefined",
  "isError",
  "isFalsy",
  "isFunction",
  "isIterable",
  "isIterator",
  "isMap",
  "isNegative",
  "isNegativeOrZero",
  "isNull",
  "isNullish",
  "isNumber",
  "isNumberValue",
  "isObject",
  "isObjectValue",
  "isObserver",
  "isPlainObject",
  "isPositive",
  "isPositiveOrZero",
  "isRegExp",
  "isSet",
  "isString",
  "isSymbol",
  "isTruthy",
  "isUndefined",
  "isWeakMap",
  "isWeakSet",
  "mapKeys",
  "mapValues",
  "negate",
  "noop",
  "not",
  "peekIterable",
  "reduceProps",
  "rethrow",
  "unwrapPromise",
  "wait",
]
`;
